realtime_ai_character/llm/__init__.py

```diff
@@ -6,19 +6,28 @@
 from realtime_ai_character.llm.base import LLM
 
 
-def get_llm(model='gpt-3.5-turbo-16k') -> LLM:
-    if model.startswith('gpt'):
-        from realtime_ai_character.llm.openai_llm import OpenaiLlm
-        return OpenaiLlm(model=model)
-    elif model.startswith('claude'):
-        from realtime_ai_character.llm.anthropic_llm import AnthropicLlm
-        return AnthropicLlm(model=model)
-    elif "llama" in model:
-        # Currently use Anyscale to support llama models
-        from realtime_ai_character.llm.anyscale_llm import AnysacleLlm
-        return AnysacleLlm(model=model)
-    else:
-        raise ValueError(f'Invalid llm model: {model}')
+def get_llm(model="gpt-3.5-turbo-16k") -> LLM:
+    # temporaryly comment out all other llms
+    # need figure out how to set up llama2wrapper in frontend
+    from realtime_ai_character.llm.llama2wrapper_llm import Llama2wrapperLlm
+
+    return Llama2wrapperLlm(url=model)
```

Comment:
let's keep the branching logic for the formal PR

Reply:
also might need a convention to route to local, e.g. maybe just call it `local` for now

Reply:
Thanks! I found that `OPENAI_API_KEY` in `.env` is always required and if not will raise the error:
```
openai.error.AuthenticationError: Incorrect API key provided: YOUR_API_KEY. You can find your API key at https://platform.openai.com/account/api-keys.
```

Reply:
> let's keep the branching logic for the formal PR

If I keep other branches here, the arg `model` will always be "gpt-3.5-turbo-16k", then initial a `OpenaiLlm`.
I think the reason is that there is no `local` button on frontend, and my choice `GPT-3.5` will always set arg `model` as `gpt-3.5-turbo-16k`.
And `LLM_MODEL_USE` from `.env` is overwritten by frontend choice.

![屏幕截图 2023-08-26 021701](https://github.com/Shaunwei/RealChar/assets/11456256/3ec78749-9332-4571-9d31-349f46c0d791)

Reply:
(presumably this is unusable on 3090 / too slow, right? ) @liltom-eth  - do you have an a100 - or 2x 4090s?

Reply:
I understand your current code makes showcasing the demo easier, but for us to merge the code base we should still aim to incorporate with existing logic. I suggest we first make the backend part ready. 

For the frontend selection, we can make an environment variable or UI advanced option to enable local Llama inference. When this is toggled, the model string passed to the backend can be your choice here in the backend.  The frontend part can be a separate PR if you would like. For testing only, you can change the model  string of the "Llama-2-70b" to test.

Reply:
> (presumably this is unusable on 3090 / too slow, right? ) @liltom-eth - do you have an a100 - or 2x 4090s?
I believe it is usable on 3090, (running gptq model 18.85 tokens/sec on 2080ti).

Reply:
> I understand your current code makes showcasing the demo easier, but for us to merge the code base we should still aim to incorporate with existing logic. I suggest we first make the backend part ready.
> 
> For the frontend selection, we can make an environment variable or UI advanced option to enable local Llama inference. When this is toggled, the model string passed to the backend can be your choice here in the backend. The frontend part can be a separate PR if you would like. For testing only, you can change the model string of the "Llama-2-70b" to test.

Thank you! I will test it by using "Llama-2-70b" button in this PR. Another PR for frontend would be helphul.

Reply:
@pycui When I tried the frontend button "Llama-2-70b", it always through an error like:

<img width="510" alt="Screenshot 2023-08-28 at 8 56 35 PM" src="https://github.com/Shaunwei/RealChar/assets/11456256/132db0a3-a102-412e-a746-feb89e1959d0">

Is that error happening because of checking anyscale key?

Reply:
It seems because using a non-3.5 model directs you to a firebase auth, but you probably don't have a working Firebase App. You can probably edit this`client/web/src/App.jsx` L218, so that (in your test) your model name doesn't require a sign-in.